src/coreclr/vm/peimagelayout.cpp

```diff
@@ -502,6 +519,7 @@ MappedImageLayout::MappedImageLayout(PEImage* pOwner)
 
     HANDLE hFile = pOwner->GetFileHandle();
     INT64 offset = pOwner->GetOffset();
+    _ASSERTE(!pOwner->GetUncompressedSize());
```

Comment:
Nit: Personally I would prefer to have a full `== 0` in this case. I hate the C++ tricks of treating numericals as booleans - it confuses me when reading the code.